test/integration/vm_translator_test.rb

```diff
@@ -3,83 +3,19 @@
 class VMTranslatorIntegrationTest < Minitest::Test
   def test_integration_test
     assembly_code = `bin/vm-translator examples/Push.vm`
-    expected = <<~EOF
-      @17
-      D=A
-      @SP
-      A=M
-      M=D
-      @SP
-      M=M+1
-    EOF
-
+    expected = File.open("test/fixtures/Push.asm").readlines.join
```

Comment:
Extracting the fixtures is a smart idea!

It’s easier to use `File.read`:

```suggestion
    expected = File.read("test/fixtures/Push.asm")
```